python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -231,6 +231,10 @@ def run(self, inputCovariances, camera=None, inputExpList=None):
                     datasetPtc.rawVars[ampName].append(partialPtcDataset.rawVars[ampName][0])
                 else:
                     datasetPtc.rawVars[ampName].append(partialPtcDataset.rawVars[ampName])
+                if type(partialPtcDataset.expIdMask[ampName]) is list:
```

Comment:
Are there any cases where these are not lists with a single element each?

Reply:
In theory, no. After running {{cpExtract}}, these should be lists with single elements.